src/main/kotlin/com/google/protobuf/gradle/ProtobufConfiguratorExts.kt

```diff
@@ -6,9 +6,7 @@ import org.gradle.api.Project
 import org.gradle.api.file.SourceDirectorySet
 import org.gradle.api.plugins.ExtensionAware
 import org.gradle.api.tasks.SourceSet
-import org.gradle.kotlin.dsl.NamedDomainObjectContainerScope
-import org.gradle.kotlin.dsl.closureOf
-import org.gradle.kotlin.dsl.get
+import org.gradle.kotlin.dsl.*
```

Comment:
Why this change?

Reply:
Auto import in IDEA. Reverted to using single import statements.